osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -116,6 +117,14 @@ private void load(OsuConfigManager config)
             if (beatmap.BeatmapInfo.AudioLeadIn > 0)
                 startTime = Math.Min(startTime, firstHitObjectTime - beatmap.BeatmapInfo.AudioLeadIn);
 
+            // some beatmaps have no AudioLeadIn but the video starts before the first object
+            var videoLayer = beatmap.Storyboard.GetLayer(LegacyStoryLayer.Video);
```

Comment:
I'm a little bit confused as to why this is here. It should be included as part of `beatmap.Storyboard.FirstEventTime` (above)?

Reply:
Sorry, I left these in because of the previous (non-storyboard) implementation, not realizing the logic was already there